src/Main.hs

```diff
@@ -18,7 +18,7 @@ import Control.Concurrent
 import Data.IORef
 
 import FuncTorrent.Bencode (decode)
-import FuncTorrent.Logger (initLogger, logMessage, logStop)
+import FuncTorrent.Logger (Log, initLogger, logMessage, logError, logStop)
```

Comment:
I'm on the edge. I don't mind either actually. Anything is ok for our own code, but I would really prefer explicit imports from libraries.